src/sentry/api/endpoints/organization_profiling_functions.py

```diff
@@ -220,6 +223,17 @@ def get_trends_data(stats_data):
             reverse=data["trend"] is TrendType.REGRESSION,
         )
 
+        if data["trend"] is TrendType.REGRESSION:
+            try:
+                if features.has(
```

Comment:
you might want to use options here if it becomes a high traffic area but it's not necessary atm

Reply:
This is in the endpoint used by the profiling landing page. Additionally, this uses the ingest feature flag which controls if the issue will be ingested or not which is backed by options.

Reply:
oh you want to create issues out of function trends. got it 👍